packages/core/bootstrap/src/lib/external-adapter/validator.ts

```diff
@@ -117,6 +168,29 @@ export class Validator {
     return multiple
   }
 
+  overrideToken = (symbol: string, network = 'ethereum'): string | undefined => {
+    if (!this.validated.tokenOverrides) return undefined
+    return this.validated.tokenOverrides.get(network.toLowerCase())?.get(symbol.toLowerCase())
+  }
+
+  overrideIncludes = (adapter: string, from: string, to: string, includes: Includes[]): Includes | undefined => {
+    includes.filter(include =>
+      (include.from.toLowerCase() === from.toLowerCase() && include.to.toLowerCase() === to.toLowerCase()) &&
+      (!include.adapters || include.adapters.map(adapter => adapter.toLowerCase()).includes(adapter.toLowerCase()))
+    )[0]
+    // Search through `presetIncludes` to find matching override for adapter and to/from pairing.
+    for (const pair of presetIncludes) {
+      if(pair.from === from && pair.to === to) {
+        for (let i = 0; i < pair.includes.length; i++) {
+          if (pair.includes[i].adapters.includes(adapter.toUpperCase())) {
+            return pair.includes[i]
+          }
+        }
+      }
+    }
```

Comment:
Instead of doing a lot of nesting, we should be able to use some lambda functions to make it a bit more readable. Below is just an (untested) suggestion:

```suggestion
    // Search through `presetIncludes` to find matching override for adapter and to/from pairing.
    const pairs = presetIncludes.filter(pair => pair.from === from && pair.to === to)
    for (const pair of pairs) {
      const matchingIncludes = pair.includes
        .find(include => include.adapters.length === 0 || include.adapters.includes(adapter.toLowerCase()))
      if (matchingIncludes) return matchingIncludes
    }
```